data/kernels/colorspace.h

```diff
@@ -629,8 +629,8 @@ static inline float4 dt_XYZ_to_xyY(const float4 XYZ)
   const float sum = XYZ.x + XYZ.y + XYZ.z;
   if(XYZ.x == 0.0f && XYZ.y == 0.0f && XYZ.z == 0.0f)
   {
-    xyY.x = 0.312700492f;
-    xyY.y = 0.329000939f;
+    xyY.x = (float)0.31271;
```

Comment:
Is there a point writing `(float)0.31271` instead of `0.31271f`

Reply:
I guess no. Emphasizing that's in fact double precision converted as at all other places.